ui/apps/analysis/src/features/flow/containers/Node/NodeTypes/CohortSelectionNode/CohortSelectionNode.tsx

```diff
@@ -24,9 +24,43 @@ export const CohortSelectionNode = (
         name={data.name}
         onSettingClick={openSetting}
         resultType={data.error ? "error" : "success"}
+        renderChildren
         node={node}
       >
-        {data.description}
+        <div style={{ padding: "8px 12px" }}>
```

Comment:
NIT: would prefer to avoid incline styles and use the stylesheet for readability